src/components/overall-statistics/tabs/PlayerActivityTab.vue

```diff
@@ -341,5 +342,32 @@ export default class PlayerActivityTab extends Vue {
   get popularGameHours(): PopularGameHour[] {
     return this.$store.direct.state.overallStatistics.popularGameHours;
   }
+
+  // Fix for chart.js displaying the graph for new modes to the left of the chart instead of to the right.
+  // Fill the gameDays array with dummy data if the gamemode hasn't existed long enough to fill up the chart. 
+  private fillDays(gameDaysObjects: GameDayPerMode[]) {
+
+    // Get the number of days from the 1vs1 mode as a reference.
+    const expectedGameDays = gameDaysObjects.find(obj => obj.gameMode == EGameMode.GM_1ON1)!.gameDays.length;
```

Comment:
probably better to either hardcode this, or make it part of the object that the backend sends
e.g.:
```{
    gameDays: 60,
    gameData: {["data goes here"]}
}```

Reply:
or better yet, make it a constant:
https://github.com/w3champions/website/blob/00961f6f45a5348f88b51f429e16d6650d8eb05c/src/constants.ts
and also have ChartJS refer to that for the number of columns to use too